src/Shouldly.Tests/ShouldContain/LongStringScenario.cs

```diff
@@ -12,7 +12,7 @@ protected override void ShouldThrowAWobbly()
 
         protected override string ChuckedAWobblyErrorMessage
         {
-            get { return "longString should contain \"fff\" but does not"; }
+            get { return "longString should contain \"fff\" but was actually \"aaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaaa\""; }
```

Comment:
Say we have:

```
var longstring = "@Shouldly is an assertion framework which focuses on giving great error messages when the assertion fails while being simple and terse.

This is the old Assert way:

Assert.That(contestant.Points, Is.EqualTo(1337));

For your troubles, you get this message, when it fails:

Expected 1337 but was 0

How it Should be:

contestant.Points.ShouldBe(1337);

Which is just syntax, so far, but check out the message when it fails:

contestant.Points should be 1337 but was 0

It might be easy to underestimate how useful this is. Another example, side by side:

Assert.That(map.IndexOfValue("boo"), Is.EqualTo(2));    // -> Expected 2 but was 1
map.IndexOfValue("boo").ShouldBe(2);                    // -> map.IndexOfValue("boo") should be 2 but was 1

Shouldly uses the code before the ShouldBe statement to report on errors, which makes diagnosing easier.

Read more about Shouldly and it's features at http://docs.shouldly-lib.net/";

longString.ShouldContain("aaaa"); // fails
```
That entire string will be dumped into the output, making it really hard to read. Having a max length on the string we put in the exception message will make it better